content/documentation/database/generators.md

```diff
@@ -9,27 +9,30 @@ aliases:
 
 ```bash
 $ buffalo pop g --help
+Generates config, model, and migrations files.
 
 Usage:
-  buffalo pop generate [command]
+  buffalo-pop pop generate [command]
```

Comment:
Actually, this is one of my concerns related to the "command or flag" discussion for help messages in the other change.

The original command the user executed was `buffalo pop ...` but the usage was printed as `buffalo-pop pop ...`, and I feel this behavior is not great even though it is still reasonable and the buffalo team or experienced users will fully understand it.

Anyway, this change itself is fine for the purpose of this PR.